apps/api/src/persistence/database/arango-connection.provider.ts

```diff
@@ -161,7 +161,11 @@ export class ArangoConnectionProvider {
 
         if (doesCollectionExist) return;
 
-        await this.#connection.createCollection(collectionName);
+        if (collectionName === 'resource_edge_connections') {
```

Comment:
We need a single source of truth for the edge collection name. Also, this if \ else feels a bit hacky. Is there a more elegant way to handle the differences between the two types of collection?

Reply:
Removed magic string.

Reply:
Added comment about the hackiness. We'll fix it when there is a second edge collection to add (e.g. tags?)